wordpress-mobile/WordPress-Android
# Add cancel pending auto-upload action on post list items

Fixes #10206  

**This PR should be merged into `master-auto-upload` after https://github.com/wordpress-mobile/WordPress-Android/pull/10342 is merged.**

This PR updates add Cancel action to items pending auto-upload.

To test:
Follow the test cases described [here](https://github.com/wordpress-mobile/WordPress-iOS/issues/12227). The `cancel` button isn't supported yet.
- Test also self-hosted and Jetpack sites - the remote-auto-save should be disabled for them

Update release notes:

- The release-notes were updated in https://github.com/wordpress-mobile/WordPress-Android/pull/10319
